From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: LessOS Build System <build@lessos>
Date: Sun, 29 Dec 2024 18:30:00 +0000
Subject: [PATCH] Add missing libraries for static parted linking

When linking statically against libparted, we need to also link
against parted's dependencies: uuid and blkid from util-linux.
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 1111111..2222222 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@ man_MANS = fatresize.1
 fatresize_CPPFLAGS = $(PARTED_CFLAGS)

 fatresize_SOURCES = fatresize.c
-fatresize_LDADD = $(PARTED_LIBS)
+fatresize_LDADD = $(PARTED_LIBS) -luuid -lblkid

 EXTRA_DIST = AUTHORS CREDITS ChangeLog NEWS README \
 	fatresize.spec $(man_MANS)
--
2.43.0
